mungegithub/mungers/approval-handler.go

```diff
@@ -90,6 +90,7 @@ func (h *ApprovalHandler) Munge(obj *github.MungeObject) {
 	}
 
 	approverSet := sets.String{}
+	approverSet.Insert(*obj.Issue.User.Name)
 
 	// from oldest to latest
 	for i := len(comments) - 1; i >= 0; i-- {
```

Comment:
Can you please maybe break it into function for readability: `CreateApproverSet() sets.String` here instead of the loop